moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -978,25 +978,30 @@ bool RobotState::hasAttachedBody(const std::string& id) const
 
 const AttachedBody* RobotState::getAttachedBody(const std::string& id) const
 {
-  std::map<std::string, AttachedBody*>::const_iterator it = attached_body_map_.find(id);
+  const auto it = attached_body_map_.find(id);
```

Comment:
Why not using a reference here?
```suggestion
  const auto& it = attached_body_map_.find(id);
```

Reply:
Indeed, that is better. Changed it.

Reply:
Sorry, I wasn't reading correctly here. Iterators, of course, shouldn't use references. They are pointer-like data structures.